### ISSUE TITLE
You can not select a working directory

### ISSUE THREAD

**Issue by soukupj** [acdvorak/intellij-lessc-plugin#64]
Not open dialog select source and output directory 
PhpStorm 7.1

![less](https://f.cloud.github.com/assets/1811943/1837939/1e64ca14-744a-11e3-8389-1e48c297f657.png)

![less-plugin](https://f.cloud.github.com/assets/1811943/1837941/27d82b4a-744a-11e3-88cc-42ad83033192.png)

![java-version](https://f.cloud.github.com/assets/1811943/1837942/2c6e8b68-744a-11e3-9aa2-b0e735c7f6f1.png)

**Comment by adam-lynch** [comment]
:+1:

**Comment by MuppetGate** [comment]
The same thing happens with IntelliJ 13. The problem seems to be caused by changes made to the IntelliJ Plugin API.

I put together a fix for IntelliJ 13. It might work for PhpStorm 7.1, but I can't guarantee it.

http://d.pr/f/nNTS

Give it a try and see :-)

Alternatively, install the Jetbrains FileWatcher plugin (though that does need the installation of NodeJS as well).

**Comment by adam-lynch** [comment]
Thanks @MuppetGate, that version works for me in IntelliJ 13.0.1 on Windows 8, but not on Mac OS X (no error message, nothing). I've had problems with other plugins not working on Mac before, is this a common/known problem?

---

**FileWatcher**

About the FileWatcher plugin, I don't think it covers everything. Take this example.
- `.../css/less/bootstrap.less`
- `.../css/less/importedIntoTheFileAboveMe.less`
- `.../css/boostrap.css`

With this plugin, I create a LESS profile, set `.../css/less/` as the LESS directory, only include `.../css/less/boostrap.less` (by absolute path), and set the output directory as `.../css/`. Therefore, a change to `importedIntoTheFileAboveMe.less` will result in only a new `.../css/bootstrap.css`.

With the FileWatcher plugin, this setup doesn't seem possible. Does anyone know if this has already been requested for the FileWatcher plugin?

For now I've resorted to using [Prepos](http://alphapixels.com/prepros/), a standalone LESS compiler (among other things). It's very fast, can watch (even like I described above) and has nice notifications that pop in from the bottom right while staying out of your way. (I'm not affiliated)
